tests/test_invoices.py

```diff
@@ -574,9 +575,9 @@ def test_create_invoice_with_many_items(client):
     # ))
 
     # testing debug panel
-    # url = reverse("debugpanel_invoice_forcepreview",
+    # url = reverse("debug_panel_invoice_forcepreview",
     #               kwargs={'invoice_id': invoice.id})
-    # url = reverse('debugpanel_invoice_placeholders')
+    # url = reverse('debug_panel_invoice_placeholders')
```

Comment:
maybe you could remove this part of commented code?

Reply:
i'd keep it here, because it serves as sort of a doc about how to use this test for design purposes. Notice that it doesn't have any assertions, it just sets everything up so you can break into it and check how it renders the results.